.env

```diff
@@ -5,4 +5,5 @@ DATABASE_USERNAME=fusionauth
 DATABASE_PASSWORD=fusionauth
 ES_JAVA_OPTS="-Xms512m -Xmx512m"
 FUSIONAUTH_APP_MEMORY=512M
-
+FUSEKI_JVM_ARGS="-Xmx6g"
+FUSEKI_DATA_DIR=/fast/scratch/fuseki
```

Comment:
I think the values that go in this file should be ones that either never have to change, or that are sensible defaults.

Can we move this value to `.envrc.example` with a suitable comment? Also, I think what you said in the PR text suggests a better value:
> Point FUSEKI_DATA_DIR to a directory like ~/apache-jena-fuseki-2.6.0/run